### PR TITLE
[POM][TopMenuPage][TopMenuSupport][SupportDropDownMenu]

### PR DESCRIPTION
**Pull request by Iriskus**

### CODE REVIEW

**Comment by Iriskus** [comment]
POM [POM][TopMenuPage][TopMenuSupport][SupportDropDownMenu] testFAQButtonIsClickable()<IrinaIrico>
https://trello.com/c/Wznj0qHi/1354-pom-test-name-github-username

**Review by verafes — APPROVED** [review]

File: src/test/java/old_tests/HappyStrawberryTest.java [hunk 1]
```diff
@@ -32,6 +33,7 @@ public void testMenuPricingIsClickable() {
 
 
     }
+    @Ignore
```
Comment (SergeiKuznetsov) [inline src/test/java/old_tests/HappyStrawberryTest.java:36]:
```suggestio

    @Ignore
```

Comment (SergeiKuznetsov) [inline src/test/java/old_tests/HappyStrawberryTest.java:36]:
```suggestion

    @Ignore
```

File: src/test/java/pages/MainPage.java [hunk 2]
```diff
@@ -135,4 +143,18 @@ public MainPage clickOnIconsOnDifferentWeatherPopUp(WebElement element) {
...
+
+        return this;
+    }
+
+    public MainPage clickFAQButton(){
+         click(buttonFAQ);
+
+        return this;
+    }
+
```
Comment (SergeiKuznetsov) [inline src/test/java/pages/MainPage.java:159]:
```suggestion
```

**Review by SergeiKuznetsov — APPROVED** [review]

**Pull request merged**